internal/commands/document.go

```diff
@@ -152,8 +152,8 @@ func getHeader(comments []string) (Header, error) {
 	var description string
 	var resources string
 	for _, comment := range comments {
-		if strings.Contains(comment, "@title") {
-			title = strings.SplitAfter(comment, "@title")[1]
+		if strings.Contains(strings.ToLower(comment), "@title") {
+			title = strings.SplitAfter(strings.ToLower(comment), "@title")[1]
```

Comment:
This change makes the entire title lowercase which I don't think was the desired outcome.

Reply:
This has been fixed.